core-bundle/templates/Backend/be_header_menu.html.twig

```diff
@@ -1,6 +1,203 @@
 {% trans_default_domain 'contao_default' %}
 <nav aria-label="{{ 'MSC.headerNavigation'|trans }}">
     <a href="{{ app.request.requestUri }}#skipHeaderNav" class="invisible">{{ 'MSC.skipNavigation'|trans }}</a>
+    {% if searchEnabled %}
+        <div class="backend-search_outer">
+            <div class="backend-search_overlay"></div>
+            <div class="backend-search_inner">
+                <form id="backend-search_form" class="backend-search_form" action="contao/search" method="GET" target="_blank">
+                    <input id="backend-search" class="tl_text" autocomplete="off" placeholder="Search..." type="text" name="keywords">
+                    <label for="backend-search">
+                        <span class="invisible">Search</span>
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search"><circle cx="11" cy="11" r="8"/><path d="m21 21-4.3-4.3"/></svg>
+                    </label>
+                </form>
+                <div class="backend-search_results">
+                    <span>Type to start searching</span>
+                </div>
+            </div>
+        </div>
+        <script>
```

Comment:
Maybe a total noob question but shouldn't this also be a Stimulus controller like all the other stuff we started to migrate? In case the entire page is re-loaded using Turbo etc.?

Reply:
@Toflar It will be- PR is in draft mode for this reason :P and that was the state of the presentation. I wanted to make sure this isn't lost